pkg/services/sqlstore/alert.go

```diff
@@ -83,7 +83,11 @@ func deleteAlertByIdInternal(alertId int64, reason string, sess *DBSession) erro
 func HandleAlertsQuery(query *models.GetAlertsQuery) error {
 	builder := SqlBuilder{}
 
-	builder.Write(`SELECT
+	joinType := "INNER"
+	if query.StandaloneAlertsEnabled {
+		joinType = "LEFT OUTER"
```

Comment:
Is this logic really needed? can't we always do an outer join?

Reply:
I guess we can. I differentiated only for keeping the existing behaviour as is and not fetching additional rows if there are not needed.